test/quantization/test_numeric_suite_fx.py

```diff
@@ -601,6 +601,35 @@ def forward(self, x0):
         }
         self.assert_types_for_matched_subgraph_pairs(results, expected_types, mp, mq)
 
+    @override_qengines
+    def test_dict_return_type(self):
+        # verify that we can traverse up nodes which return dictionaries
+        class M(nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, x0):
+                x1 = torch.add(x0, 1.0)
+                y1 = torch.add(x0, 1.0)
+                z1 = {'x1': x1, 'y1': (y1,)}
```

Comment:
Looks good, since you have recursive function for adding lists/dicts, do you also want to test list[list] or dict[list]?